app/views/plans/index.html.erb

```diff
@@ -1,104 +1,105 @@
-<% javascript "views/plans/index.js" %>
-<h1><%= _('My Dashboard') %></h1>
-
-<!-- if the user has projects -->
-<p class="left-indent">
-<% if @plans.count > 0 %>
+<div class="row">
+  <div class="col-md-12">    
+    <h1><%= _('My Dashboard') %></h1>
 
-    <%= _('The table below lists the plans that you have created, and any that have been shared with you by others.') %><br />
-    <%= _('These can be edited, shared, exported or deleted at anytime.')%>
-<% else %>
-    <strong><%= _("Welcome.") %></strong><br />
-    <%= _("You are now ready to create your first DMP.") %><br />
-    <%= _("Click the 'Create plan' button below to begin.")%>
-<% end %>
-</p>
+    <!-- if the user has projects -->
+    <p>
+      <% if @plans.count > 0 %>
+        <%= _('The table below lists the plans that you have created, and any that have been shared with you by others.') %><br />
+        <%= _('These can be edited, shared, exported or deleted at anytime.')%>
+      <% else %>
+        <strong><%= _("Welcome.") %></strong><br />
+        <%= _("You are now ready to create your first DMP.") %><br />
+        <%= _("Click the 'Create plan' button below to begin.")%>
+      <% end %>
+    </p>
+  </div>
+</div>
 
 <div class="row">
   <div class="col-md-12">
     <% if @plans.length > 0 %>
-      <table class="dmp_table tablesorter">
-        <thead>
-          <% if @plans.length > 10 %>
+      <div class="table-responsive">
+        <table class="table table-hover tablesorter">
+          <thead>
+            <% if @plans.length > 10 %>

```

Comment:
It was discussed in a quick conversation. I agree that it should be a constant to make it easily changeable in one spot, will move to `config/initializers/constants`.